docs/import/manage_disaster.js

```diff
@@ -255,30 +246,16 @@ function onSetDisaster() {
   scoreBoundsMap.initialize(
       scoreBoundsPath ? transformGeoPointArrayToLatLng(scoreBoundsPath) : null,
       states);
-  const neededStates = [];
-  for (const state of states) {
-    if (!stateAssets.has(state)) {
-      neededStates.push(state);
-    }
-  }
-  if (neededStates) {
-    const newStatePromises = getStatesAssetsFromEe(neededStates);
-    for (const [state, promise] of newStatePromises) {
-      stateAssets.set(state, promise);
-    }
-  }
-  const statePromises = [];
-  for (const state of states) {
-    statePromises.push(stateAssets.get(state));
-  }
-  const scorePromise = Promise.all(statePromises).then((stateAssets) => {
-    if (getDisaster() === currentDisaster &&
-        !processedCurrentDisasterStateAssets) {
-      // Don't do anything unless this is still the right disaster.
-      initializeScoreSelectors(states, stateAssets);
-      processedCurrentDisasterStateAssets = true;
-    }
-  });
+  // getStateAssetsFromEe does internal caching.
```

Comment:
does this mean we don't need disasterAssets either because that also does internal cacheing? If so can we leave a TODO?

Reply:
Great point, TODO added.

Reply:
Might be nice to leave similar TODO in manage_layers but understand if you don't want to touch that file in this PR. Maybe could leave a general clean up callers TODO in list_ee_assets. Though I already have a TODO so unify manage_layers and manage_disasters so

Reply:
I promise to do them together :)

Reply:
#397